bin/oneoff/rescore_taught_in_past

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+# This script relies on the existence of an empty file '/tmp/rescored_applications' to write to.
+
+ActiveRecord::Base.transaction do
+  rescored_applications = []
+  puts "Rescoring taught_in_past for teacher 1920 applications..."
+  Pd::Application::Teacher1920Application.find_each do |application|
+    responses = application.sanitize_form_data_hash
+    options = application.class.options
+    response_scores = JSON.parse(application.response_scores)
+
+    old_taught_in_past_score = response_scores["bonus_points_scores"] ? response_scores["bonus_points_scores"]["taught_in_past"] : nil
+    new_taught_in_past_score = (responses[:taught_in_past] & options[:taught_in_past].values_at(3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 15)).any? ? 0 : 2
```

Comment:
nit: this should probably also use `TAUGHT_IN_PAST_NO_BONUS_POINT_RESPONSES`, although fragility for a one-off script is much less frightening